.gitlab-ci.yml

```diff
@@ -58,9 +58,10 @@ flake8-diff:
 
 shellcheck:
   stage: build
-  image: pipelinecomponents/shellcheck:0.5.4
-  script:
+  image: koalaman/shellcheck-alpine:v0.7.1
```

Comment:
Can you explain (e.g. in the commit message) why you choose this one?

Reply:
Different versions of `shellcheck` report different issues for some code. So @MuteSpirit  choose the version  he used.